tests/__snapshots__/null-list.spec.ts.snap

```diff
@@ -608,333 +608,333 @@ exports[`wrapping for args; nonNullDefault = true; wrapped arg def nullable(list
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; nexus def nullable(nullable(nullable(Foo))) 1`] = `"Error: Cannot wrap a nullable() in a nullable()"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; nexus def nullable.nullable.nullable.field({type: Foo}) 1`] = `"Error: Expected [object Object] to be a valid output type, saw GraphQLInputObjectType"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(String) 1`] = `"[String!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.field({type: String}) 1`] = `"[String!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(list(String)) 1`] = `"[[String!]!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.list.field({type: String}) 1`] = `"[[String!]!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(list(list(String))) 1`] = `"[[[String!]!]!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.list.list.field({type: String}) 1`] = `"[[[String!]!]!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(list(nonNull(String))) 1`] = `"[[String!]!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.list.nonNull.field({type: String}) 1`] = `"[[String!]!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(list(nullable(String))) 1`] = `"[[String!]!]"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.list.nullable.field({type: String}) 1`] = `"[[String]!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(nonNull(String)) 1`] = `"[String!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.nonNull.field({type: String}) 1`] = `"[String!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(nonNull(list(String))) 1`] = `"[[String!]!]!"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.nonNull.list.field({type: String}) 1`] = `"[[String!]!]!"`;
 
-exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list(nonNull(nonNull(String))) 1`] = `"Error: Cannot wrap a nonNull() in a nonNull()"`;
+exports[`wrapping for input types with chained API; nonNullDefault = false; string ref list.nonNull.nonNull.field({type: String}) 1`] = `"Error: Expected String! to be a GraphQL nullable type."`;
```

Comment:
Something looks wrong here. This shouldn't happen this we're making `nonNull.nonNull` a no-op now

Reply:
Investigated the reason. That's because we're not preventing multiple `NonNull` or `Null` to be unshifted to the `wrapping` list in the definition blocks. I have a simple fix but I'm wondering whether you kept it that way intentionally. I'll let you decide.

Reply:
Hmm. Yeah it’s not intended you should ever do this, but yeah I missed this case, because the wrapping array gets applied at the end. What fix were you thinking?

Reply:
That was my fix 👇 

![image](https://user-images.githubusercontent.com/25233379/99970303-e8921280-2d9b-11eb-91cc-eca41578292d.png)